machine.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict, Set
+
+from coin import Coin
+from costumer import Costumer
+from drink import Drink
+from resource import Resource
+
+
+def print_coins(coins: Dict[Coin, int]):
```

Comment:
I think the fact that this function just sits here implies there should be some code responsible for the coin-conversion thingy. I mean, if you had money or coins object you could easily and cleanly print it and query it's value, etc..